arkane/statmech.py

```diff
@@ -59,6 +59,7 @@
 from arkane.molpro import MolproLog
 from arkane.output import prettify
 from arkane.qchem import QChemLog
+from arkane.orca import OrcaLog
```

Comment:
We sort imports alphabetically, could you relocate the orca import and put it after the molpro one?